codec.go

```diff
@@ -0,0 +1,66 @@
+package ccloud
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+
+	"github.com/gogo/protobuf/jsonpb"
+	"github.com/gogo/protobuf/proto"
+	codec1978 "github.com/ugorji/go/codec"
+)
+
+// hack for dep so it doesn't remove the dependency
+var _ = codec1978.GenVersion
```

Comment:
Hmm, I wonder why the dependency is needed. I assume this is just copied/pasted though, so it's fine to keep it as is.

Reply:
yeah this is all copied..